components/gfx/display_list/mod.rs

```diff
@@ -701,16 +705,16 @@ pub struct IframeDisplayItem {
 #[derive(Clone, Deserialize, MallocSizeOf, Serialize)]
 pub struct Gradient {
     /// The start point of the gradient (computed during display list construction).
-    pub start_point: Point2D<Au>,
+    pub start_point: LayoutPoint,
 
     /// The end point of the gradient (computed during display list construction).
-    pub end_point: Point2D<Au>,
+    pub end_point: LayoutPoint,
 
     /// A list of color stops.
-    pub stops: Vec<GradientStop>,
+    pub stops: Vec<webrender_api::GradientStop>,
 
-    /// True if gradient repeats infinitly.
-    pub repeating: bool,
+    /// Whether the gradient is repeated or clamped.
+    pub extend: webrender_api::ExtendMode,
```

Comment:
Same comment here and other places in the file as well. I would also call this `extend_mode` in order to make the code more readable.